src/components/Card/Card.jsx

```diff
@@ -6,8 +6,6 @@ import './Card.css';
 
 const Card = (props) => {
```

Comment:
I wanna to suggest you make your code better.
Can you please destructurize props here and pass only one method for card updating.
After that changes you can implement any function you want for specified updating of some card's value.

Reply:
done